WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/JPScreenshotTest.java

```diff
@@ -211,19 +466,15 @@ private boolean runningInTestLab() {
         return "true".equals(testLabSetting);
     }
 
-    // In some cases there's a gray overlay on view pager screens when taking screenshots
-    // this function swipes left and then right as a workaround to clear it
-    // resourceID should be the ID of the viewPager
-    private void swipeToAvoidGrayOverlay(int resourceID) {
-        // Workaround to avoid gray overlay
-        swipeLeftOnViewPager(resourceID);
-        idleFor(1000);
-        swipeRightOnViewPager(resourceID);
-        idleFor(1000);
+    private void checkAndNavigateToSiteIfNotShowing() {
```

Comment:
This is another great enhancement that makes the work on the screenshots much easier now that we can order the tests in any way and easily disable the ones we're not working on while in development or debugging 👍 .

Thank you for the effort to make this possible @zwarm  🥇